docker/docker-compose.dev.yml

```diff
@@ -69,6 +69,7 @@ services:
         target: /app/entrypoint.sh
     environment:
       TEST_SANCTIONS_CONTRACT: 'mock'
+      WHITELISTING: ${WHITELISTING}
```

Comment:
It is better to move this env var to the `docker/docker-compose.yml` file, since this one is exclusively for tests/dev.

Reply:
Normally I would agree, however, that will make code that defaults to WHITELISTING=disable and somewhat invalidates the point of this change, which is to ensure that if people turn of this important security feature, they do so knowingly.

Reply:
As discussed.  `WHITELISTING` is now disabled in dev mode so that people don't need to remember to set then environment variable.